Coverity workflow: Configure fails with `None of the required 'libsodium' found`

Looking at the [workflow Coverity #108][1], configure fails:

```
-- Checking for one of the modules 'libsodium'
CMake Error at /usr/local/share/cmake-3.20/Modules/FindPkgConfig.cmake:843 (message):
  None of the required 'libsodium' found
Call Stack (most recent call first):
  src/Cedar/CMakeLists.txt:35 (pkg_search_module)
[…]
 -- Configuring incomplete, errors occurred!
See also "/home/runner/work/SoftEtherVPN/SoftEtherVPN/build/CMakeFiles/CMakeOutput.log".
See also "/home/runner/work/SoftEtherVPN/SoftEtherVPN/build/CMakeFiles/CMakeError.log".
```

The `|| true` in the workflow configuration covers this issue up.

https://github.com/SoftEtherVPN/SoftEtherVPN/blob/192083e7c46ccf8155238eecccd222cad6c94d38/.github/workflows/coverity.yml#L26-L30

[1]: https://github.com/SoftEtherVPN/SoftEtherVPN/runs/2815671030?check_suite_focus=true

--- a/.github/workflows/coverity.yml
+++ b/.github/workflows/coverity.yml
@@ -22,7 +22,7 @@ jobs:
     - name: Install apt dependencies
       run: |
         sudo apt-get update
-        sudo apt-get install -y cmake gcc g++ libncurses5-dev libreadline-dev libssl-dev make zlib1g-dev
+        sudo apt-get install -y cmake gcc g++ libncurses5-dev libreadline-dev libssl-dev make zlib1g-dev libsodium-dev
     - name: Run Coverity Scan
       env:
         COVERITY_SCAN_TOKEN: ${{ secrets.COVERITY_SCAN_TOKEN }}

